pkg/azure/azure.go

```diff
@@ -0,0 +1,55 @@
+package azure
+
+import (
+	"context"
+	"log/slog"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/grafana/cloudcost-exporter/pkg/provider"
+)
+
+const (
+	// subsystem = "azure"
+	collectTimeout time.Duration = 5 * time.Minute
+)
+
+var (
+// TODO - add prometheus metrics here
+)
+
+type Azure struct {
+	Context context.Context
+	Logger  *slog.Logger
+}
+
+type Config struct {
+	Logger *slog.Logger
+}
+
+// New is a TODO
+func New(ctx context.Context, config *Config) (*Azure, error) {
+	providerGroup := config.Logger.WithGroup("azure")
```

Comment:
Suggestion: Make `azure` refer to the `subsystem` const that's been commented out

Reply:
done